chapter_computer-vision/image-augmentation.md

```diff
@@ -262,7 +262,12 @@ def train_batch_ch13(net, features, labels, loss, trainer, devices,
 #@tab pytorch
 #@save
 def train_batch_ch13(net, X, y, loss, trainer, devices):
-    X, y = X.to(devices[0]), y.to(devices[0])
+    if isinstance(X, list):
+        # Required for BERT Finetuning (Sec 15.7)
+        X = [inp.to(devices[0])for inp in X]
```

Comment:
space before `for`
`inp` is not clear
Required for BERT Fine-tuning (to be covered later)

Reply:
maybe `inp` -> 'x'?